practices/Git.md

```diff
@@ -0,0 +1,110 @@
+---
+slug: git
+stages:
+  - development
+short_description: Git is a distributed version control system (VCS) tool used for tracking source code during development. Utilizing Git improves collboration among fellow developers.
+tags:
+  - git 
+  - branching
+  - Data Model
+  - Index
+  - VCS
+  - pull requests
+  - good developer experience
+keywords:
+  - git
+  - gitflow workflow
+  - git flow
+  - github
+  - branching 
+  - data model
+  - commit
+  - index
+  - head
+  - tree
+  - blob
+  - push
+  - pull
+---
+
+# Git 
+
+**TL;DR**
+
+Git is a distributed version control system (VCS) tool used for tracking source code during development. Utilizing Git improves collaboration among fellow developers.
+
+## What Is Git?
+
+Git is one of many VCS that are there in the market but it simply is the most popular among the developers. It allows flawless management for large and small projects. The basic difference between Git and other VCS is mainly the way they store their files. Generally, VCS other than Git store files in the form of a list as file-based changes (this is known as delta-version control system). 
+But Git on the other hand saves the data in the form of snapshots of the file-system. Whenever you commit some changes to the project it will take the snapshot and save it with a reference and for efficiency, it does not take the snapshot if the file is not changed it just links it to the previous ones.
+It was first created by [Linus Torvalds](https://en.wikipedia.org/wiki/Linus_Torvalds) in 2005 for the development of the [Linux kernel](https://en.wikipedia.org/wiki/Linux_kernel).
+
+
+## How Does Git Work?
```

Comment:
The second headline should be `Why You Might Want the %{name}`. See [Contribution guide](https://github.com/DXHeroes/knowledge-base-content/blob/master/CONTRIBUTING.md).

Please, change the level to `###`.

```suggestion
### How Does Git Work?
```